azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -94,97 +71,67 @@ def configure_azure_monitor(**kwargs) -> None:
     disable_logging = configurations[DISABLE_LOGGING_ARG]
     disable_metrics = configurations[DISABLE_METRICS_ARG]
 
-    resource = None
-    if not disable_logging or not disable_tracing or not disable_metrics:
-        resource = _get_resource(configurations)
-
     # Setup tracing pipeline
     if not disable_tracing:
-        _setup_tracing(resource, configurations)
+        _setup_tracing(configurations)
 
     # Setup logging pipeline
     if not disable_logging:
-        _setup_logging(resource, configurations)
+        _setup_logging(configurations)
 
     # Setup metrics pipeline
     if not disable_metrics:
-        _setup_metrics(resource, configurations)
+        _setup_metrics(configurations)
 
     # Setup instrumentations
     # Instrumentations need to be setup last so to use the global providers
     # instanstiated in the other setup steps
-    _setup_instrumentations(configurations)
-
-
-def _get_resource(configurations: Dict[str, ConfigurationValue]) -> Resource:
-    return configurations.get(RESOURCE_ARG, Resource.create())
+    _setup_instrumentations()
 
 
-def _setup_tracing(
-    resource: Resource, configurations: Dict[str, ConfigurationValue]
-):
+def _setup_tracing(configurations: Dict[str, ConfigurationValue]):
     sampling_ratio = configurations[SAMPLING_RATIO_ARG]
-    tracing_export_interval_ms = configurations[TRACING_EXPORT_INTERVAL_MS_ARG]
     tracer_provider = TracerProvider(
         sampler=ApplicationInsightsSampler(sampling_ratio=sampling_ratio),
-        resource=resource,
     )
     set_tracer_provider(tracer_provider)
     trace_exporter = AzureMonitorTraceExporter(**configurations)
     span_processor = BatchSpanProcessor(
         trace_exporter,
-        schedule_delay_millis=tracing_export_interval_ms,
     )
     get_tracer_provider().add_span_processor(span_processor)
 
 
-def _setup_logging(
-    resource: Resource, configurations: Dict[str, ConfigurationValue]
-):
-    logger_name = configurations[LOGGER_NAME_ARG]
-    logging_level = configurations[LOGGING_LEVEL_ARG]
+def _setup_logging(configurations: Dict[str, ConfigurationValue]):
+    # TODO: Remove after upgrading to 1.18
```

Comment:
Does this mean upgrading to opentelemetry-sdk 1.18 because it's in the latest release?

Reply:
Yeah

Reply:
Specified